c8run/cmd/c8run/main.go

```diff
@@ -176,7 +179,7 @@ func createStartFlagSet(settings *types.C8RunSettings) *flag.FlagSet {
 	startFlagSet.StringVar(&settings.Keystore, "keystore", "", "Provide a JKS filepath to enable TLS")
 	startFlagSet.StringVar(&settings.KeystorePassword, "keystorePassword", "", "Provide a password to unlock your JKS keystore")
 	startFlagSet.StringVar(&settings.LogLevel, "log-level", "", "Adjust the log level of Camunda")
-	startFlagSet.BoolVar(&settings.DisableElasticsearch, "disable-elasticsearch", false, "Do not start or stop Elasticsearch (still requires Elasticsearch to be running outside of c8run)")
+	startFlagSet.BoolVar(&settings.DisableElasticsearch, "disable-elasticsearch", true, "Do not start or stop Elasticsearch (disabling will enable H2)")
```

Comment:
We need to think what we do with this flag now. It was used prior to disable the startup of Elasticsearch to let users connect to an external Elasticsearch.

Once the RDBMS is default this flag can be removed as all database configuration should be only set in the applciation.yaml. 

How could this work now for the intermediate phase? (run ES per default; allowing for external ES; allowing for RDBMS config)?

Reply:
We have the following use cases:
- Start internal ES by default
- being able to connect to external ES
- being able to start RDBMS